test/js/third_party/body-parser/express-body-parser-test.test.ts

```diff
@@ -21,6 +21,35 @@ test("iconv works", () => {
   expect(iconv.encodingExists("us-ascii")).toBe(true);
 });
 
+// https://github.com/oven-sh/bun/issues/7031
+test("doesn't reuse listeners", async () => {
+  const PORT = 9999;
```

Comment:
Instead of a hard-coded port, use port `0` to create the first server. Then use the `.address()` getter to get the port, then use that to listen the 2nd time.

Reply:
https://github.com/oven-sh/bun/pull/7126/commits/975dbb1b88654fc4bbf54cb9131da1c210244715